### PR TITLE
grey out dashboard ipv6

### LINKED ISSUE #833: Gray out ipv6 on Dashboard to match card/row view

### PR DESCRIPTION
**Pull request by na3d**
Closes #833

### CODE REVIEW

File: src/linodes/linode/layouts/DashboardPage.js [hunk 1]
```diff
@@ -176,7 +176,7 @@ export class DashboardPage extends Component {
               <div className="col-sm-9 content-col right">
                 <ul className="list-unstyled">
                   <li>{linode.ipv4}</li>
-                  <li className="text-light-black">{linode.ipv6}</li>
+                  <li className="linode-ipv6">{linode.ipv6}</li>
```
Comment (eatonphil) [inline src/linodes/linode/layouts/DashboardPage.js]: There's also a text-lightest-black class. I'd go with that since it's shorter and this page needs heavy refactoring to come.

Comment (na3d) [inline src/linodes/linode/layouts/DashboardPage.js]: 👍

Comment (na3d) [inline src/linodes/linode/layouts/DashboardPage.js]: Funny enough, text-lightest-black isn't working, and light-black never really did anything.

Comment (mparke) [inline src/linodes/linode/layouts/DashboardPage.js]: use the bootstrap `text-muted` class, and apply `color: $lightest-black;` in scss for `text-muted`

Comment (eatonphil) [inline src/linodes/linode/layouts/DashboardPage.js]: text-muted is too specific here. I'd keep it how the commit currently is.

Comment (mparke) [inline src/linodes/linode/layouts/DashboardPage.js]: text-muted is purposely generic @eatonphil

Comment (eatonphil) [inline src/linodes/linode/layouts/DashboardPage.js]: Well we have a global text-muted class that is too specific. :)

Comment (na3d) [inline src/linodes/linode/layouts/DashboardPage.js]: So text-muted on the dashboard, and also on the initial page for the cards and rows?

It doesn't look bad like that.

Comment (na3d) [inline src/linodes/linode/layouts/DashboardPage.js]: screen shot 2017-01-09 at 2 02 14 pm

Comment (eatonphil) [inline src/linodes/linode/layouts/DashboardPage.js]: No, because we don't want that font size here.

Comment (mparke) [inline src/linodes/linode/layouts/DashboardPage.js]: ah, @eatonphil I see:

```
.text-muted {
    font-size: $font-xs-small;
}
```,

so let's fix that now, 

find and apply this font size to existing places ( there doesn't look to be many ) by defining the font-size for `<small>` or creating a new class, 

and then use `.text-muted` only for our color, `$lightest-black`

Comment (eatonphil) [inline src/linodes/linode/layouts/DashboardPage.js]: Well now we've gotten into scope creep and this is not my task. If @na3d would like to do that and make a new PR with that change then that's up to him. Otherwise I'm fine with this PR for now.

Comment (mparke) [inline src/linodes/linode/layouts/DashboardPage.js]: I don't believe this to be scope creep if it's directly related to the issue. This is sort of a broken window example, where we could leave it, but we're gonna run into this again, so why not correct it/remove any ambiguities now?

Comment (na3d) [inline src/linodes/linode/layouts/DashboardPage.js]: There are only a couple places where text-muted is used.  I can fix it the right way real quick.

**Comment by na3d** [comment]
screen shot 2017-01-09 at 3 23 23 pm

File: scss/manager.scss [hunk 2]
```diff
@@ -843,6 +834,9 @@ header .sub-header {
 
 .list-unstyled {
     margin-bottom: 0;
+    .linode-ipv6 {
+        color: $lightest-black;
+    }
```
Comment (eatonphil) [inline scss/manager.scss]: Is this getting dropped?

Comment (na3d) [inline scss/manager.scss]: I thought about removing that so much that I thought I did it already.

Comment (na3d) [inline scss/manager.scss]: Fixed with this one.
